documentation/modules/managing/con-producer-config-properties.adoc

```diff
@@ -223,7 +230,7 @@ When your application calls `KafkaProducer.send()` the messages are:
 * Added to a batch of messages in a per-partition queue
 
 At which point the `send()` method returns.
-So the time `send()` is blocked is determined by:
+So the time `send()` is blocked is determined by the following:
```

Comment:
The send() operation also blocks when buffer.memory is full or metadata are not available. We should also mention these cases for completeness.